web/admin.py

```
@@ -101,14 +101,32 @@ class WaitingRoomAdmin(admin.ModelAdmin):
 @admin.register(Profile)
 class ProfileAdmin(admin.ModelAdmin):
-    list_display = ("user", "is_teacher", "expertise", "created_at", "updated_at")
-    list_filter = ("is_teacher", "created_at", "updated_at")
-    search_fields = ("user__username", "user__email", "expertise", "bio")
+    list_display = (
+        "user", 
+        "is_teacher", 
+        "points",  # NEW: Add points to list view
+        "expertise", 
+        "created_at", 
+        "updated_at"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)